lib/frontmatter-linter.js

```diff
@@ -23,23 +18,33 @@ const rules = {
       )}" (See https://github.com/emberjs/rfcs#stages)`,
     },
   },
-  'Start Date': {
+  'start-date': {
     type: 'date',
     required: true,
     allowEmpty: false,
     messages: {
       type: 'must be a date formatted YYYY-MM-DD',
     },
   },
-  'Relevant Team(s)': {
-    type: 'string',
+  teams: {
     required: true,
-    allowEmpty: false,
+    conform: function (value) {
+      if (!value) {
+        // it's ok for teams to be empty
```

Comment:
Why is it ok for teams to be empty?

Reply:
well that's a good question 😂 this is mainly because of the fact that there are currently a bunch of RFCs that aren't assigned to a team. I would **much** prefer to remove this ability and just update the teams on all the existing PRs

Reply:
What do you think about setting all the RFCs that don't currently have a team to just be part of the framework team?

Reply:
also I've updated the tests in this PR now but left the teams question as a failure until we resolve this conversation 👍

Reply:
i've fixed the teams to always be required again 👍